src/tasks/SystemsManagerRunCommand/task.json

```
@@ -227,6 +232,24 @@
             "helpMarkDown": "The name of a variable that will contain the unique ID assigned to the command. The command ID can be used future references to the request.",
             "required": false
         },
+        {
+            "name": "cloudWatchOutputEnabled",
+            "label": "Enable CloudWatch Logs",
+            "type": "boolean",
+            "required": false,
+            "defaultValue": false,
+            "groupName": "cloudWatchOutput",
+            "helpMarkDown": "Enables Systems Manager to send command output to CloudWatch Logs."
+        },
+        {
+            "name": "cloudWatchLogGroupName",
+            "label": "CloudWatch Log Group Name",
+            "type": "string",
+            "required": false,
+            "defaultValue": "",
+            "groupName": "cloudWatchOutput",
+            "helpMarkDown": "The name of the CloudWatch log group where you want to send command output. If you don’t specify a group name, Systems Manager automatically creates a log group for you. The log group uses the following naming format: <em>aws/ssm/SystemsManagerDocumentName</em> ."
```

Nit: Assuming that the default log group name replaces "SystemsManagerDocumentName"? If so, replace with `<em>aws/ssm/<systems_manager_document_name></em>`
```suggestion
            "helpMarkDown": "The name of the CloudWatch log group where you want to send command output. If you don’t specify a group name, Systems Manager automatically creates a log group for you. The log group uses the following naming format: <em>aws/ssm/<systems_manager_document_name></em>."
```

---

It really makes sense. I just copied that from AWS CLI docs, and yes, it might be confusing. 
https://awscli.amazonaws.com/v2/documentation/api/latest/reference/ssm/send-command.html